### DISCUSSION TITLE
Question about TGOV1 model

### DISCUSSION THREAD

**Discussion by zhui99** [CURENT/andes#464] [Q&A] [answered]
Dear Dr.Cui,
I tried to add a regcp1 model on bus 8 in ieee14 system. I did not add any toggle and alter before 1s, but the TDS result looks like abnormal from 0-1s。
![image](https://github.com/CURENT/andes/assets/138122314/7dc125c9-95b7-44c4-9e04-acf1d8792f51)
Then，I replaced the TGOV1 model to HYGOV model on bus 8, the frequecy is stable from 0-1s.
![image](https://github.com/CURENT/andes/assets/138122314/2c1207f7-c83f-43b3-a1a3-3f2b42801c33)
I would like to ask for the reason. Two case files are attached.
[ieee14_regcp1_hygov.xlsx](https://github.com/CURENT/andes/files/11905517/ieee14_regcp1_hygov.xlsx)
[ieee14_regcp1_tgov1.xlsx](https://github.com/CURENT/andes/files/11905518/ieee14_regcp1_tgov1.xlsx)

**Comment by jinningwang** [comment, marked as answer]
In Figure 1, it looks like the TDS initialization failed because the system should be in a steady state without disturbance. You can check the TDS initialization status with ``TDS.init()`` in Case 1. This can be caused by inappropriate limiter values introduced by REGCP1.

There are some discussions related to failed TDS initialization, they might be helpful for you.

**Reply by zhui99** [reply]
Thanks for your answer! I have found the error of the case1.
